lib/rhc/deployment_helpers.rb

```diff
@@ -9,15 +9,17 @@ module DeploymentHelpers
 
     protected
 
-      def deploy_artifact(rest_app, artifact, hot_deploy, force_clean_build)
```

Comment:
Because `deploy_artifact` is a method of `RHC::DeploymentHelpers`, which is used only as a mixin in `RHC::Commands::App` and `RHC::Commands::Deployment`, both of which subclass `RHC::Commands::Base`, which defines `options`, therefore the `options` parameter here is redundant (I think).